supervisor/shared/reload.c

```diff
@@ -82,15 +82,15 @@ inline bool autoreload_is_enabled() {
 }
 
 void autoreload_trigger() {
-    if (autoreload_enabled & !autoreload_suspended) {
-        last_autoreload_trigger = supervisor_ticks_ms32();
-        // Guard against the rare time that ticks is 0;
```

Comment:
Why did you remove this bit?

Reply:
You mean "Guard against the rare time that ticks is 0"?
Do we want to trigger auto reload when ticks is 0? Will there even be such a case?